### DISCUSSION TITLE
Top Menu 04 (Optional) more icons ?

### DISCUSSION THREAD

**Discussion by cucpo** [Blackymas/NSPanel_HA_Blueprint#438] [General]
Hi.
It is possible to display more icons in the top row, as shown in the picture on project page ?

**Comment by lloydbayley** [comment]
Apparently it's coming in the next version. Configurable 'chips'.

**Comment by Blackymas** [comment]
Yes maybe tomorrow 🤩

**Comment by Blackymas** [comment]
Is now online 😂🤣
